spring-integration-core/src/main/java/org/springframework/integration/util/MessagingMethodInvokerHelper.java

```diff
@@ -979,7 +979,7 @@ private void setExclusiveTargetParameterType(TypeDescriptor targetParameterType,
 		}
 	}
 
-	private static class ParametersWrapper {
+	public static class ParametersWrapper {
```

Comment:
One of these days, both IDEs will catch up with each other and we won't have to taunt each other with these comments 😄